service/raft/raft_group_registry.cc

```diff
@@ -86,7 +164,8 @@ void raft_group_registry::init_rpc_verbs() {
             auto& rpc = self.get_rpc(gid);
             // The address learnt from a probably unknown server should
             // eventually expire
-            self._srv_address_mappings.set(from, std::move(addr), true);
+            rslog.debug("Adding address {} -> {} to the address map", from, addr);
```

Comment:
I was using the DEBUG level for `rslog` to debug stuff in the past. But this will make the DEBUG level too crowded for `rslog` (this is printed on **every** received message) and I'll never want to enable it again. I think this should be `trace` at most, or even not happen at all. Is it that useful?

Reply:
I can drop it. If you can enable debug level log you can add extra logging to the source code.

Reply:
Removed.